akka-docs/rst/java/stream/stream-quickstart.rst

```diff
@@ -184,7 +184,7 @@ more advanced graph elements to finally be consumed by a ``Sink<In,M3>``.
 
 The first type parameter—:class:`Tweet` in this case—designates the kind of elements produced
 by the source while the ``M`` type parameters describe the object that is created during
-materialization (:ref:`see below <materialized-values-quick-java>`)—:class:`BoxedUnit` (from the ``scala.runtime``
+materialization (:ref:`see below <materialized-values-quick-java>`)—:class:`NotUsed` (from the ``scala.runtime``
```

Comment:
[This](http://doc.akka.io/docs/akka/2.4/java/stream/stream-quickstart.html#Broadcasting_a_stream) refers to [TwitterStreamQuickstartDocTest.java](https://github.com/akka/akka/blob/4a7cd147313404eb8137e56163e54f06c23640d0/akka-docs/rst/java/code/docs/stream/TwitterStreamQuickstartDocTest.java#L231)

```
    final Source<Author, NotUsed> authors =
```

So, `NotUsed` is correct.